lib/model.js

```diff
@@ -2533,6 +2533,7 @@ class Model {
    * @param  {boolean}        [options.benchmark=false]        Pass query execution time in milliseconds as second argument to logging function (options.logging).
    * @param  {boolean|Array}  [options.returning=false]        If true, append RETURNING <model columns> to get back all defined values; if an array of column names, append RETURNING <columns> to get back specific columns (Postgres only)
    * @param  {string}         [options.searchPath=DEFAULT]     An optional parameter to specify the schema search_path (Postgres only)
+   * @param  {Array}          [options.upsertKeys]             Fields to use as the unique keys (on duplicate key update)? (only supported by MySQL, MariaDB, SQLite >= 3.24.0 & Postgres >= 9.5). By default, unique keys and/or primary keys are used.
```

Comment:
I don't see anything in the MySQL or MariaDB docs which show that this is actually supported.

They only have `ON DUPLICATE KEY` whereas for postgres and sqlite this is translated to `ON CONFLICT ({upsertKeys}) DO UPDATE ...`.